{liboop,lsh}: drop (#334495)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2411.section.md

- The `services.trust-dns` module has been renamed to `services.hickory-dns`.

> - The `services.lsh` module has been remove as the corresponding package had no maintainer in Nixpkgs and hasn't seen an upstream release in over a decade. It is recommended to migrate to `services.openss`

Reviewer: ```suggestion
- The `lsh` package and the `services.lsh` module have been removed as they had no maintainer in Nixpkgs and hadn’t seen an upstream release in over a decade. It is recommended to migrate to `openssh` and `services.openssh`.
```

> - The `lsh` package and the `services.lsh` module have been removed as they had no maintainer in Nixpkgs and hadn’t seen an upstream release in over a decade. It is recommended to migrate to `openssh` and `services.openssh`.

Reviewer: Ah, sorry, this time *I* messed up:
```suggestion
- The `lsh` package and the `services.lshd` module have been removed as they had no maintainer in Nixpkgs and hadn’t seen an upstream release in over a decade. It is recommended to migrate to `openssh` and `services.openssh`.
```
